lib/request/util.js

```diff
@@ -0,0 +1,20 @@
+var curl2postman = require('curl-to-postmanv2'),
+    Collection = require('postman-collection').Collection;
+
+const convert = function (value, callback) {
+    let collection = new Collection();
+
+    curl2postman.convert({ type: 'string', data: value }, function (err, result) {
+        if (err) {
+            return callback(err);
+        }
+        collection.items.add({
+            name: result.output[0] && result.output[0].data.name,
+            request: result.output[0] && result.output[0].data
+        });
+    });
+
+    return collection;
```

Comment:
Even though the method `curl2postman.convert` is synchronous in execution nature but its implementation has been done in a way of the async method by the module owner and we will have to follow that. So, we can't return the `collection` from here.

We need to move this statement inside the callback of `curl2postman.convert` and return the collection via callback. For example `return callback(null, collection)`, `null` for no-error.